### ISSUE TITLE
SCRUM - PipelinePeakcallingScrum

### ISSUE THREAD

**Issue by Charlie-George** [CGATOxford/CGATPipelines#240]
Alot of the MACS2 output is renamed/reformatted - is this required? it makes it very hard to track expected macs2 output should be compared to what is documented in the macs2 manual

**Comment by snsansom** [comment]
Hi Charlie, 

I just submitted some fixes to PipelinePeakcalling to get macs2 to work for a ChIP-seq project (https://github.com/CGATOxford/CGATPipelines/pull/243) and then noticed this comment...

In general the purpose of this pipeline compared to pipeline_chipseq is perhaps a little unclear. Is it correct that the standard CGAT workflow for ChIP-seq analysis now pipeline_peakcalling + pipeline_intervals and that pipeline_chipseq is essentially depreciated? What are people using for ATAC-seq?

I would always argue for minimal post-processing of the output of algorithms such as MACS2, but I suspect a lot of the report depends upon the renaming/reformatting. Otherwise I know some of the reformatting is done for very valid space considerations - e.g. the conversion from bedgraph to bigwig.

**Comment by Charlie-George** [comment]
Thanks Steve that really helps! 

Your right the standard CGAT workflow at the moment is  pipeline_peakcalling + pipeline_intervals and pipeline_chipseq is redundant - however Katy and I are working on a new peakcalling_pipeline (from scratch in terms of the pipeline structure, but using bits of the old code) at the moment that will incorporate IDR analysis as an option into the code and streamline the peakcalling pipeline bit to make it more flexible in terms of bam filtering and also remove some of the unnecessary complexity of the current pipeline. The purpose of this pipeline will be solely to take a bam and generate a set of high confidence peaks along with some QC metrics - further analysis/peak exploration will be done elsewhere (intervals - diff_bind etc).

In terms of ATAC-Seq I just made a small project-specific pipeline as I wanted to filter my bams based on some extra parameters like MAPQ and collect some extra stats that the current pipeline didn't allow.

**Comment by snsansom** [comment]
Thanks Charlie, this is extremely useful to know! 

Very glad this key workflow is getting the attention it deserves! Look forward to seeing the new focused version - its a good idea to incorporate IDR and it would also be awesome if it can incorporate your code for ATAC-seq analysis!

(One other thing we noticed with the current pipeline is that the syntax for tagging control samples is not well documented - I figured it out by looking at one of Andreas' old projects.)

**Comment by Charlie-George** [comment]
Thanks Steve, hopefully it will be up and working soon. Your right about the syntax for the inputs being cryptic - in the new pipeline we thought it would be better to have a design table where the correct input for each sample is stated explicitly as some of our projects have inconsistencies in the input matching between samples - this design would make sure the user was responsible for checking the inputs were correctly assigned and is also compatible for several ChIP-Seq packages in R that are quite useful such as DiffBind and ChIPQC.

On 9 Aug 2016, at 14:11, Steve Sansom <notifications@github.com<mailto:notifications@github.com>> wrote:

Thanks Charlie, this is extremely useful to know!

Very glad this key workflow is getting the attention it deserves! Look forward to seeing the new focused version - its a good idea to incorporate IDR and it would also be awesome if can incorporate your code for ATAC-seq analysis!

(One other thing we noticed with the currently pipeline is that the syntax for tagging control samples is not well documented - I figured it out by looking at one of Andreas' old projects.)
